share/nb.po

```diff
@@ -167,6 +167,10 @@ msgstr ""
 "installert på dette system?\n"
 "\n"
 
+#, perl-brace-format
+msgid "Address {address} cannot be used as source address for DNS queries.\n"
+msgstr "Address {address} cannot be used as source address for DNS queries.\n"
+
```

Comment:
This should also be excluded for the same reason.

Reply:
@matsduf As you asked for something different (but it was a long time ago), do you think that @mattias-p proposal is reasonable ?

Reply:
That is fine. I am not sure if it should be empty or non-existent, but at least the latter will mean that msgid will shown, and then when you translate you will be prompted.